README.md

```diff
@@ -1,23 +1,59 @@
 [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A523.04.3-brightgreen.svg)](https://www.nextflow.io/)
 
-# Example Pipeline for IRIDA Next
+# Genomic Address Service Nomenclature Workflow
 
-This is an example pipeline to be used for integration with IRIDA Next.
+This workflow takes provided JSON-formatted MLST allelic profiles and assigns cluster addresses to samples based on an existing cluster designations. This pipeline is designed to be integrated into IRIDA Next. However, it may be run as a stand-alone pipeline.
+
+A brief overview of the usage of this pipeline is given below. Detailed documentation can be found in the [docs/](docs/) directory.
 
 # Input
 
 The input to the pipeline is a standard sample sheet (passed as `--input samplesheet.csv`) that looks like:
 
-| sample  | fastq_1         | fastq_2         |
-| ------- | --------------- | --------------- |
-| SampleA | file_1.fastq.gz | file_2.fastq.gz |
+| sample  | mlst_alleles      | address |
+| ------- | ----------------- | ------- |
+| sampleA | sampleA.mlst.json | 1.1.1   |
+| sampleQ | sampleQ.mlst.json |         |
+| sampleF | sampleF.mlst.json |         |
 
 The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/).
 
+Details on the columns can be found in the [Full samplesheet](docs/usage.md#full-samplesheet) documentation.
+
 # Parameters
 
 The main parameters are `--input` as defined above and `--output` for specifying the output results directory. You may wish to provide `-profile singularity` to specify the use of singularity containers and `-r [branch]` to specify which GitHub branch you would like to run.
 
+## Profile dists
+
+The following can be used to adjust parameters for the [profile_dists][] tool.
+
+- `--pd_outfmt`: The output format for distances. For this pipeline the only valid value is _pairwise_ (required by [gas call][]).
```

Comment:
I think I remember this coming up at one point, but if there's only one option, why the parameter?

Reply:
That's a good point. We can remove the parameter in another PR. I had kept `--pd_outfmt` in the gasclustering pipeline for now in case there was changes to profile_dists/gas, but probably best to remove the `--pd_outfmt` parameter until that is the case.

Reply:
`--pd_outfmt` parameter has been removed here: [04ee4fd](https://github.com/phac-nml/gasnomenclature/pull/12/commits/04ee4fd8044719d6d76a2c786266337af7d829b3)

And the README was updated here: [a1b3685](https://github.com/phac-nml/gasnomenclature/pull/12/commits/a1b36852f018497b8fd50144b7031a1ae7f97876)